modules/ui/src/Form/StockTransactions1.php

```diff
@@ -43,14 +43,6 @@ public function buildForm(array $form, FormStateInterface $form_state) {
   /**
    * {@inheritdoc}
    */
-  public function validateForm(array &$form, FormStateInterface $form_state) {
```

Comment:
Yepp, sure. I have done that and created a PR against @steveoliver s branch. But he seems somewhat bussy, so he didn't get that in. Will ping him, to get that done.